packages/storybook/stories/va-accordion-uswds.stories.jsx

```diff
@@ -23,49 +23,49 @@ export default {
 
 const Template = args => (
   <va-accordion {...args}>
-    <va-accordion-item id="first" header="First Amendment">
+    <va-accordion-item id="first" header="First Amendment" uswds={args.uswds}>
```

Comment:
`uswds` should default to `true` now so do you need to add this the USWDS stories?

Reply:
Oh, you're right... I added this because of the toggle in the properties table

Reply:
Removed from other stories. Did you want me to remove this and the default as well?

Reply:
> I added this because of the toggle in the properties table

That's a good observation! I think since we're removing that `uswds` prop soon anyway (when the v1 version is removed), it might save us some cleanup to just keep it out for now. And sure, if you want to take it out of `defaultArgs`, I think that's fine to do as well.

Reply:
On it!